configs/config.json.example

```diff
@@ -153,6 +153,7 @@
         "type": "EvolvePokemon",
         "config": {
           "enabled": false,
+          "interval": 60,

```

Comment:
this number represents how often - in seconds - the task runs. I just realized I should also add that to the docs.